libs/journeys/ui/src/components/RadioQuestion/GridVariant/GridVariant.spec.tsx

```
@@ -0,0 +1,47 @@
+import { GridVariant as RadioOptionGridVariant } from '../../RadioOption/GridVariant'
+
+import { GridVariant } from './GridVariant'
+
+describe('GridVariant', () => {
+  const mockHandleClick = jest.fn()
+  const options = [
+    <RadioOptionGridVariant
+      key="option1"
+      label="Option 1"
+      handleClick={mockHandleClick}
+      children={[]}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)